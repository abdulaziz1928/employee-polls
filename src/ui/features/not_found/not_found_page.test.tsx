import { render } from "@testing-library/react";
import NotFoundPage from "./not_found_page";

describe("NotFoundPage", () => {
  it("should render the component", () => {
    const { container } = render(<NotFoundPage />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
