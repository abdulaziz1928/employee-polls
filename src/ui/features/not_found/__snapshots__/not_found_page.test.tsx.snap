// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render the component 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1tyumyi-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-0"
    >
      <h1
        class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter css-47suzj-MuiTypography-root"
      >
        404 Not Found
      </h1>
      <h2
        class="MuiTypography-root MuiTypography-h4 MuiTypography-alignCenter MuiTypography-gutterBottom css-m2bl7s-MuiTypography-root"
      >
        The page you are looking for doesn't exist
      </h2>
    </div>
  </div>
</div>
`;
